src/TLuaInterpreter.cpp

```diff
@@ -429,12 +430,12 @@ int TLuaInterpreter::getProfileName( lua_State * L )
     return 1;
 }
 
-int TLuaInterpreter::raiseInterProfileEvent( lua_State * L )
+int TLuaInterpreter::sendGlobalEvent( lua_State * L )
```

Comment:
I think I might have suggested this as a much simpler name, however, for consistency how would you feel about raiseGlobalEvent intead?